pymatgen/analysis/tests/test_phase_diagram.py

```diff
@@ -323,14 +351,14 @@ def test_get_quasi_e_to_hull(self):
         stable_entry = [e for e in self.pd.stable_entries if e.name == "Li2O"][0]
 
         self.assertEqual(
-            self.pd.get_quasi_e_to_hull(duplicate_entry),
-            self.pd.get_quasi_e_to_hull(stable_entry),
+            self.pd.get_phase_separation_energy(duplicate_entry),
+            self.pd.get_phase_separation_energy(stable_entry),
             "Novel duplicates of stable entries should have same decomposition energy!",
         )
```

Comment:
This test was to check that the hashing work around worked. I have left the test in but it doesn't test anything that shouldn't already be covered by another test.